controllers/volumereplication_controller.go

```diff
@@ -42,6 +45,8 @@ import (
 
 const (
 	pvcDataSource = "PersistentVolumeClaim"
+	VolumeReplicationClass = "VolumeReplicationClass"
+	VolumeReplication = "VolumeReplication"
 )
```

Comment:
variables should be unexported should start with smallcase

Reply:
done